src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodosContext } from '../../contexts/TodoProvider';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { selectTodo: onTodoSelect, selectedTodo } = useContext(TodosContext);
+  const { completed, title } = todo;
+
+  return (
+    <tr data-cy="todo">
+      <td className="is-vcentered">
+        {todo.id}
+      </td>
+
+      <td className="is-vcentered">
+        {completed
+          && (
+            <span className="icon" data-cy="iconCompleted">
+              <i className="fas fa-check" />
+            </span>
+          )}
```

Comment:
```suggestion
        {!!completed
          && (
            <span className="icon" data-cy="iconCompleted">
              <i className="fas fa-check" />
            </span>
          )}
```
in rendering with && for best rendering control use !! to return a boolean value

Reply:
but this variable already has boolean type